### PR TITLE
feat: select retro type

### LINKED ISSUE #675: [FEATURE]: select retro type
## Summary

When the user clicks on "Add new board" button, a page with the the options: Regular Retro and SPLIT retro must be rendered as per figma's design. When the option SPLIT retro is selected, the user is redirected to the page "Create new SPLIT board" (already exists). If the option Regular Retro is selected, then the user is redirected to a new menu, that also contains two options: "Configure Board" and "Quick Board" (check figma's design).

## User story it relates to

#674 

### Basic example

image
image

### PR DESCRIPTION
**Pull request by CatiaAntunes96**
Relates to #675

## Screenshots (if visual changes)
image
image

## Proposed Changes

  - Now, in the `New Board `page the user can select one of two options
  - If he selects the SPLIT retro option, is redirected to the `NewSplitBoard` page, which contains the old content of the New  Board page.
  - If he selects the Regular retro, is redirected to the `NewRegularBoard` page, where he can select on of the two options.

This pull request closes #675

### CODE REVIEW

File: frontend/src/components/CreateBoard/SelectBoardType/BoxRowContainer/styles.tsx [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+const StyledBox = styled(Flex, Box, {
+  width: '510px',
+  pt: '$32',
+  pb: '$32',
+  px: '$32',
+  borderRadius: '$12',
+  background: 'white',
+  '&:hover': {
+    cursor: 'pointer',
+  },
```
Comment (nunocaseiro) [inline frontend/src/components/CreateBoard/SelectBoardType/BoxRowContainer/styles.tsx:14]: I think the background colour black is missed when the user hovers the box. The opposite of the text colour

File: frontend/src/pages/boards/newSplitBoard.tsx [hunk 2]
```diff
@@ -0,0 +1,281 @@
...
+  /**
+   * Team  Hook
+   */
+  const {
+    fetchTeamsOfUser: { data: teamsData },
+  } = useTeam({ autoFetchTeam: false });
+
+  const {
+    fetchAllTeams: { data: allTeamsData },
+  } = useTeam({ autoFetchTeam: false });
```
Comment (nunocaseiro) [inline frontend/src/pages/boards/newSplitBoard.tsx:103]: Here we need to discuss if it makes sense to fetch all the teams or only those the user can create a split board. this means if the user is a team admin or stakeholder

**Review by nunocaseiro — CHANGES_REQUESTED** [review]
Amazing job 🚀

File: frontend/src/pages/boards/newSplitBoard.tsx (hunk 2 above)
Comment (CatiaAntunes96) [reply frontend/src/pages/boards/newSplitBoard.tsx:103]: This condition will be dealt with in issue #677, in which you can only create team boards if the user is a team admin or team stakeholder. However, if the user is super admin, he should be able to create a board from any team right? This situation is not contemplated in the design.

**Review by nunocaseiro — APPROVED** [review]
LGTM :rocket:

**Pull request merged**